src/main/resources/theme/style.css

```diff
@@ -21,6 +21,9 @@
   -fx-font-family: 'Open Sans';
   -fx-font-size: 14px;
   -fx-disabled-opacity: 0.8;
+  /*Rating changed text Colour*/
+  -winner-label-colour: -swatch-green-700;
+  -loser-label-colour: -swatch-red-700;
```

Comment:
We're using en_US, so it's `color` in the CSS classes and in the comment :) (also notice that English nouns are lower case).
Having a comment explaining what those classes are for indicates that the class names are not self-explanatory. Also, "winner" and "loser" imply that someone won or lost, but this is not what the rating change is about. So how about:

```
-rating-change-gain-color
-rating-change-loss-color
```